py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -66,10 +66,11 @@ def __init__(
         self.builder = trt.Builder(self.logger)
 
         flag = 0
-
-        # It is deprecated to not use this flag
-        EXPLICIT_BATCH = 1 << (int)(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH)
-        flag |= EXPLICIT_BATCH
+        if compilation_settings.use_strong_types:
+            STRONGLY_TYPED = 1 << (int)(
+                trt.NetworkDefinitionCreationFlag.STRONGLY_TYPED
+            )
+            flag |= STRONGLY_TYPED
```

Comment:
Does this need change in ops like below?
https://github.com/pytorch/TensorRT/pull/3111/files#diff-89deb3709966060e627cf17441fb7d47a72cb12faf484da6c6b968f9c72b2811R37-R55
In my test with sd_unet model, I got error from IConvolutionLayer/INormalizationLayer/IConcatenationLayer when strongly typed network is used. If multiple input is used for layer, trt expected same dtype when strongly typed network is enabled.